spec/web/authorize_spec.rb

```diff
@@ -70,11 +69,22 @@
   context "rails form helper" do
     before do
       @transaction = Datatrans::Web::Transaction.new(@datatrans, @valid_params)
-      @view = ActionView::Base.new
+
+      if Gem.loaded_specs['activesupport'].version >= Gem::Version.create('6.0')
+        @view = ActionView::Base.new(ActionController::Base.view_paths, {}, {})
+      else
+        @view = ActionView::Base.new
+      end
     end
 
-    it "should generate valid form field string" do
-      expect(@view.datatrans_notification_request_hidden_fields(@datatrans, @transaction)).to eq "<input type=\"hidden\" name=\"merchantId\" id=\"merchantId\" value=\"1100000000\" /><input type=\"hidden\" name=\"hiddenMode\" id=\"hiddenMode\" value=\"yes\" /><input type=\"hidden\" name=\"reqtype\" id=\"reqtype\" value=\"NOA\" /><input type=\"hidden\" name=\"amount\" id=\"amount\" value=\"1000\" /><input type=\"hidden\" name=\"currency\" id=\"currency\" value=\"CHF\" /><input type=\"hidden\" name=\"useAlias\" id=\"useAlias\" value=\"yes\" /><input type=\"hidden\" name=\"sign\" id=\"sign\" value=\"0402fb3fba8c6fcb40df9b7756e7e637\" /><input type=\"hidden\" name=\"refno\" id=\"refno\" value=\"ABCDEF\" /><input type=\"hidden\" name=\"uppCustomerDetails\" id=\"uppCustomerDetails\" /><input type=\"hidden\" name=\"uppCustomerEmail\" id=\"uppCustomerEmail\" value=\"customer@email.com\" />"
+    it 'should generate valid form field string' do
+      if Gem.loaded_specs['activesupport'].version >= Gem::Version.create('7.0')
```

Comment:
Maybe in the next major version we could force `autocomplete=off` in the gem to minimise the differences between versions.